app/main/lib/windows.ts

```diff
@@ -107,7 +109,10 @@ export function createBreakWindows(): void {
       },
     });
 
-    breakWindow.setVisibleOnAllWorkspaces(true);
+    breakWindow.setVisibleOnAllWorkspaces(true, { visibleOnFullScreen: true });
+    breakWindow.setAlwaysOnTop(true, "normal");
```

Comment:
@tom-james-watson , this is creating a bug. Removing the "normal" option fixes it. (Tested on Mac)

Reply:
fixed.